### ISSUE TITLE
RequestError: Timeout awaiting 'request' for 2000ms

### ISSUE THREAD

**Issue by tiendq** [saleor/saleor-cli#110]
I just installed `saleor-cli` and tried `saleor info`, it threw below error after 3-5 tries and finally worked. It looks like timeout error is not handled properly.

Output (truncated) with a long `TimeoutError` exception object:

```
https://saleor.io/
https://cloud.saleor.io/
https://github.com/saleor/
node:internal/process/promises:246
          triggerUncaughtException(err, true /* fromPromise */);
          ^

RequestError: Timeout awaiting 'request' for 2000ms
...
```

Saleor Commerce CLI v1.0.3 
Node v16.13.1

**Comment by zaiste** [comment]
@tiendq thanks, we are investigating it

**Comment by zaiste** [comment]
We refined the infrastructure receiving HTTP requests. The probability of timeouts should be significantly lower now. It's available as `1.2.0`.
